sdk/confidentialledger/azure-confidentialledger/azure/confidentialledger/_operations/_patch.py

```diff
@@ -0,0 +1,226 @@
+        elif polling is False:
+            polling_method = cast(PollingMethod, NoPolling())
+        else:
+            polling_method = polling
+
+        return LROPoller(self._client, initial_response, None, polling_method)
+
+    def begin_get_receipt(self, transaction_id: str, **kwargs: Any) -> LROPoller[JSON]:
+        polling = kwargs.pop("polling", True)  # type: Union[bool, PollingMethod]
+        lro_delay = kwargs.pop("polling_interval", 0.5)
+
+        def operation() -> JSON:
+            return super(ConfidentialLedgerClientOperationsMixin, self).get_receipt(
+                transaction_id=transaction_id, **kwargs
+            )
+
+        initial_response = operation()
+
+        if polling is True:
+            polling_method = cast(PollingMethod, StatePollingMethod(operation, "Ready", lro_delay))
+        elif polling is False:
+            polling_method = cast(PollingMethod, NoPolling())
+        else:
+            polling_method = polling
+
+        return LROPoller(self._client, initial_response, None, polling_method)
+
+    def begin_post_ledger_entry(
+        self,
+        entry: Union[JSON, IO],
+        *,
+        collection_id: Optional[str] = None,
+        **kwargs: Any,
+    ) -> LROPoller[JSON]:
+        """Writes a ledger entry and returns a poller to wait for it to be durably committed.
+
+        A collection id may optionally be specified.
+        """
+
+        # Pop arguments that are unexpected in the pipeline.
+        polling = kwargs.pop("polling", True)  # type: Union[bool, PollingMethod]
+        lro_delay = kwargs.pop("polling_interval", 0.5)
+
+        post_result = self.post_ledger_entry(entry, collection_id=collection_id, **kwargs)
+        transaction_id = post_result["transactionId"]
+
+        kwargs["polling"] = polling
+        kwargs["polling_interval"] = lro_delay
+        kwargs["_post_ledger_entry_response"] = post_result
+        return self.begin_wait_for_commit(transaction_id, **kwargs)
+
+    def begin_wait_for_commit(
+        self,
+        transaction_id,  # type: str
+        **kwargs,  # type: Any
+    ) -> LROPoller[JSON]:
+        """Creates a poller that queries the state of the specified transaction until it is
+        Committed, a state that indicates the transaction is durably stored in the Confidential
+        Ledger.
+        """
+        polling = kwargs.pop("polling", True)  # type: Union[bool, PollingMethod]
+        lro_delay = kwargs.pop("polling_interval", 0.5)
+
+        # If this poller was called from begin_post_ledger_entry, we should return the
+        # post_ledger_entry response, not the transaction status.
+        post_result = kwargs.pop("_post_ledger_entry_response", None)
+        deserialization_callback = lambda x: x if post_result is None else post_result
+
+        def operation() -> JSON:
+            return super(ConfidentialLedgerClientOperationsMixin, self).get_transaction_status(
```

Comment:
you can just do `super().get_transaction_status(...)`

Reply:
Wait, turns out you can't do this! Because `operation` is a nested method it doesn't have a `super`, so that's why I passed the type/object explicitly.